tutorials/process_per_core_translation/cpu_process_per_core_translation.py

```diff
@@ -0,0 +1,129 @@
+# Describtion: This script is used for CPU Multi-instance Translate, which process per core translation.
+# It can greatly speedup translate perefomance.
+# FileName: cpu_process_per_core_translation.py
+# usage: cpu_process_per_core_translation.py -m model-name -i file_to_translate -o result_to_save --batch-size 32
+# Version: 2.0
+
+import argparse
+import subprocess
+import threading
+import tempfile
+import logging
+import string
+import os
+import time
+import mxnet as mx
+
+def add_args(parser):
+    parser.add_argument("--module" , '-m', help='the pretrained module', default="wmt_model")
+    parser.add_argument("--input_file", '-i', help='the file input to translate', default='newstest2016.tc.BPE.de')
+    parser.add_argument("--output_file", '-o', help='the file saved the result of translation', default='newstest2016.tc.BPE.en')
+    parser.add_argument("--batch_size", '-bs', help='the batch size of process', default=32)
+    parser.add_argument("--data_type", '-t', help='if uses run benchmark or not', default="benchmark")
+
+def task(args):
+    os.system(args)
+
+# benchmark: multi-instances translating, each instance trans the same input file separately
```

Comment:
could you make this a proper doc-string for the method?
```
def benchmark(...):
   """
    doc goes here
   """
```

Also applies for the other functions below.

Reply:
Good style. will follow.